src/components/HeaderAuthOptions/HeaderSignUp/HeaderSignUp.js

```diff
@@ -43,7 +119,6 @@ class HeaderSignUp extends Component {
       .then(() => history.push('/first-signin'))
       .catch(error => {
         console.error(error)
-        this.setState({ email: '', username: '', password: '', passwordConfirmation: '' })
```

Comment:
This cleared the form after an error. I removed it so the user has an opportunity to alter their inputs.